wscript

```diff
@@ -309,9 +309,11 @@ def _write_version_header(tsk):
 
 
 def build(bld):
-    config_hash = Utils.h_file(bld.bldnode.make_node('ap_config.h').abspath())
-    bld.env.CCDEPS = config_hash
-    bld.env.CXXDEPS = config_hash
+    config_node = bld.bldnode.make_node('ap_config.h').abspath()
+    if os.path.exists(config_node):
+        config_hash = Utils.h_file(config_node)
+        bld.env.CCDEPS = config_hash
+        bld.env.CXXDEPS = config_hash
```

Comment:
Thanks for reporting this. This should be the correct way to solve that: https://github.com/ArduPilot/ardupilot/pull/4572